scipy/stats/tests/test_continuous.py

```diff
@@ -1050,6 +1083,7 @@ def test_rv_generic(self, i, distdata):
             'nchypergeom_fisher',     # distribution functions don't accept NaN
             'nchypergeom_wallenius',  # distribution functions don't accept NaN
             'skellam',                # during `entropy`, Fatal Python error: Aborted!
+            'zipfian',                # during init, value error due to unexpected nans
```

Comment:
This seems to have occurred because I changed `get_numerical_endpoints` to always return pairs of `float`s. I think its fixable by putting the right casts in the class generated by `make_distribution`, but just excluding for now.

Reply:
https://github.com/mdhaber/scipy/pull/121#discussion_r2060744427